<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSubValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_play">
       <property name="maximumSize">
        <size>
         <width>66</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MySlider" name="m_pSlider">
       <property name="styleSheet">
        <string notr="true">QSlider {
    background-color: rgba(22, 22, 22, 0.7);
	padding-left: 5px;
    padding-right: 5px;
	border-radius: 5px;
    /*background:transparent;*/
}

QSlider::add-page:horizontal {
	background-color: #FF7826;
	height:5px;
	border-radius: 2px;
}

QSlider::sub-page:horizontal {
	background-color: #7A7B79;
	height:5px;
	border-radius: 2px;
}

QSlider::groove:horizontal {
	background:transparent;
	height:6px;
}

QSlider::handle:horizontal {
	height: 14px;
	width: 14px;
	margin: -4px 0px -4px 0px;
	border-radius: 7px;
	background: white;
}</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="value">
        <number>33</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header>myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
